runtime/calamari/src/weights/pallet_parachain_staking.rs

```diff
@@ -352,74 +353,74 @@ impl<T: frame_system::Config> pallet_parachain_staking::WeightInfo for Substrate
     // Storage: ParachainStaking SelectedCandidates (r:0 w:1)
     // Storage: ParachainStaking DelayedPayouts (r:0 w:1)
     fn round_transition_on_initialize(x: u32, _y: u32, ) -> Weight {
-        (1_694_504_000 as Weight)
-            // Standard Error: 328_000
-            .saturating_add((2_649_000 as Weight).saturating_mul(x as Weight))
-            .saturating_add(T::DbWeight::get().reads(128 as Weight))
+        (1_507_092_000 as Weight)
+            // Standard Error: 360_000
+            .saturating_add((3_465_000 as Weight).saturating_mul(x as Weight))
+            .saturating_add(T::DbWeight::get().reads(119 as Weight))
```

Comment:
Strange that the  comments above the benchmark don't update with the change with the change in number of reads/writes.

Maybe hbs template changed

Reply:
Good catch, the hbs did not change, the comment is supposed to be output by benchmarking and there's a placeholder for it in https://github.com/Manta-Network/Manta/pull/994/files/9d3e5c3edd88249f5a00ddb97b4474ff0a82ace7#diff-389cfbb10787c5a52d8648dc621afb724f6323d9fa65d4fddf5063e5dabb540cL53

Not sure why it doesn''t actually work

Reply:
Resolving as the comment isn't terribly important and I expect this to get autofixed with some of the next deps upgrades